### DISCUSSION TITLE
How much ram is needed to use `--index-sats`?

### DISCUSSION THREAD

**Discussion by Michaelmint** [ordinals/ord#3332] [Q&A] [answered]
I am trying to index with sats however I am hitting the memory limit of system. I am working on testnet (Could that be the issue?). I have `16gb` of ram on my system and for the first 500,000 it sits at around `2gb` of usage, then it crash's at 790,000 due to me hitting my OOM. I saw a some info about needing at least 64 gb, but I am unsure if that is correct for the modern ord. Info on this would be great.

Context: https://github.com/ordinals/ord/discussions/1629

**Comment by raphjaph** [comment, marked as answer]
On mainnet you need at least 64gb. you can also play around with the `--index-cache-size` and `--commit-interval` flags and see what works for your system. Do `ord help` for more infos
